docs/controls/UniformGrid.md

```diff
@@ -180,7 +180,7 @@ In addition to `FlowDirection` to layout items from Right-to-Left.  The UWP Unif
 
 ## Requirements (Windows 10 Device Family)
 
-| [Device family](http://go.microsoft.com/fwlink/p/?LinkID=526370) | Universal, 10.0.16299.0 or higher |
+| [Device family](/windows/uwp/get-started/universal-application-platform-guide) | Universal, 10.0.16299.0 or higher |
```

Comment:
I've seen a few the FW links get switch to raw URLS, worried about them changing location in the future, that's why we kept the redirectable ones from the platform error messages.

Is this an issue with the automated tool resolving them before updating the content?